product_taxes_group/model/tax_group.py

```diff
@@ -0,0 +1,213 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Product - Taxes Group module for Odoo
+#    Copyright (C) 2014 -Today GRAP (http://www.grap.coop)
+#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import logging
+
+from openerp.osv import fields, osv
+from openerp.osv.orm import Model
+from openerp import SUPERUSER_ID
+from openerp.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+
+
+class tax_group(Model):
```

Comment:
Arf, sorry, I didn't notice. By the way, are you planning to migrate your package to 8.0 ? In the mean time, you should at least move get_or_create and init after the _columns section.  This would ease side-by-side review with account_product_fiscal_classification.

I guess, should the two package converge, they could share at least a same base class here.

By the way, having no clue on OCA community usages, could you explain me this story about thumbs up / thumbs down.
If I put a thumb up in my comment, does it mean that I vote for its inclusion in OCA packages or what ?

Regards